### PR TITLE
Validate content_sets.yml against schema

### PR DESCRIPTION
**Pull request by chmeliik**
* OSBS-5282

Signed-off-by: Adam Cmiel <acmiel@redhat.com>

Maintainers will complete the following section:
- [x] Commit messages are descriptive enough
- [x] "Signed-off-by:" line is present in each commit
- [x] Code coverage from testing does not decrease and new code is covered
- [x] JSON/YAML configuration changes are updated in the relevant schema
- [x] Changes to metadata also update the documentation for the metadata
- [x] Pull request includes link to an osbs-docs PR for user documentation updates
- [x] New feature can be disabled from a configuration file

### CODE REVIEW

File: atomic_reactor/schemas/content_sets.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+
+  "title": "Content sets container definition",
+
+  "type": ["object", "null"],
+  "patternProperties": {
+    "^[a-z].*$": {
```
Comment (chmeliik) [inline atomic_reactor/schemas/content_sets.json]: Do we want a more strict pattern here? like `[a-z][a-z0-9_]*`?

Comment (athos-ribeiro) [reply atomic_reactor/schemas/content_sets.json]: If we are not enumerating the available platforms, I don't see a reason to be more strict here

**Comment by athos-ribeiro** [comment]
Do we want a PR to the docs to have the content sets section to point to this schema?

**Comment by athos-ribeiro** [comment]
Waiving feature toggle requirement: This is the expected new behaviour for whenever this file is read

**Comment by chmeliik** [comment]
Docs PR: https://github.com/containerbuildsystem/osbs-docs/pull/129

**Pull request merged**